app/src/main/java/crux/bphc/cms/fragments/CourseSectionFragment.java

```diff
@@ -259,17 +260,22 @@ public void onClick(View view) {
                 getActivity().overridePendingTransition(R.anim.slide_in_right, R.anim.slide_out_left);
             }
         });*/
+        section.setSummary("Dear Student,<br /><br />As per Prof. YVD Rao's request, I agreed to exchange F103 with F208. <br /><br />So students who were previously assigned F103 are moved F208, F102 and F105. Please go through the <a href=\"https://docs.google.com/spreadsheets/d/1NoVh3kw7UyKHL4Y407EY_SFJZ9oprglHO_cqbYnlF5U/edit?usp=sharing\"><b>Google Spreadsheet</b></a> to see the changes.<br /><br />Sorry for the last minute change.<br /><br />Best<br />Santosh<br />");
```

Comment:
You missed this, which seems to be hardcoded for testing purposes.

Reply:
Noob mistake. Thank God this was caught.